### PR TITLE
Apply readme generator

### LINKED ISSUE #244: Refactor config structs to make a proper use of README generator
**Feature description**
Currently we have a README file generator, but we can't use it right now, cause structs need to be refactored first. The main thing to do is to get rid of the anonymous structs - all structs should have dedicated named types. After that we may generate README's for all the config files. Also adding a Makefile target would be useful for that

### PR DESCRIPTION
**Pull request by Darkren**
Did you run `make format && make check`?
Yes
Fixes #244 

 Changes:	
- README.md files were generated for visor, setup node and hypervisor configs
- Added `make generate` command to generate mocks and config readme files (requires `readmegen` to be installed with `go install`

How to test this PR:
1) Install `readmegen` with `go install` from services
2) Run `make generate`

### CODE REVIEW

**Review by evanlinjin — APPROVED** [review]
Looks good. Now we need to add the actual documentation.

File: pkg/app/appnet/mock_networker.go [hunk 1]
```diff
@@ -2,12 +2,9 @@
 
 package appnet
 
-import (
-	context "context"
-	net "net"
-
-	mock "github.com/stretchr/testify/mock"
-)
+import context "context"
```
Comment (nkryuchkov) [inline pkg/app/appnet/mock_networker.go]: Could you please format imports running `make format`?

File: pkg/app/appserver/mock_proc_manager.go [hunk 2]
```diff
@@ -2,15 +2,10 @@
...
-import (
-	io "io"
-
-	appcommon "github.com/SkycoinProject/skywire-mainnet/pkg/app/appcommon"
-
-	logging "github.com/SkycoinProject/skycoin/src/util/logging"
-
-	mock "github.com/stretchr/testify/mock"
-)
+import appcommon "github.com/SkycoinProject/skywire-mainnet/pkg/app/appcommon"
```
Comment (nkryuchkov) [inline pkg/app/appserver/mock_proc_manager.go]: Could you please format imports running make format?

File: pkg/app/mock_rpc_client.go [hunk 3]
```diff
@@ -2,15 +2,10 @@
...
-import (
-	mock "github.com/stretchr/testify/mock"
-
-	appnet "github.com/SkycoinProject/skywire-mainnet/pkg/app/appnet"
-
-	routing "github.com/SkycoinProject/skywire-mainnet/pkg/routing"
-
-	time "time"
-)
+import appnet "github.com/SkycoinProject/skywire-mainnet/pkg/app/appnet"
```
Comment (nkryuchkov) [inline pkg/app/mock_rpc_client.go]: Could you please format imports running make format?

File: pkg/router/mock_router.go [hunk 4]
```diff
@@ -2,15 +2,10 @@
...
-import (
-	context "context"
-
-	cipher "github.com/SkycoinProject/dmsg/cipher"
-
-	mock "github.com/stretchr/testify/mock"
-
-	routing "github.com/SkycoinProject/skywire-mainnet/pkg/routing"
-)
+import cipher "github.com/SkycoinProject/dmsg/cipher"
```
Comment (nkryuchkov) [inline pkg/router/mock_router.go]: Could you please format imports running make format?

File: pkg/hypervisor/README.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
+# Config
+
+- `public_key` ([PubKey](#PubKey))
```
Comment (nkryuchkov) [inline pkg/hypervisor/README.md]: Some anchors like this one seem broken. I'm not sure if it's possible to create correct anchors for such types. If not, what do you think about removing anchors for such types?

**Review by nkryuchkov — COMMENTED** [review]
Great job!

**Review by nkryuchkov — APPROVED** [review]

**Pull request merged**